Week2/homework/maartjes_work.js

```diff
@@ -1,47 +1,84 @@
-'use strict';
+"use strict";
 
+// 2.2
 const monday = [
   {
-    name: 'Write a summary HTML/CSS',
+    name: "Write a summary HTML/CSS",
     duration: 180
   },
   {
-    name: 'Some web development',
+    name: "Some web development",
     duration: 120
   },
   {
-    name: 'Fix homework for class10',
+    name: "Fix homework for class10",
     duration: 20
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   }
 ];
 
 const tuesday = [
   {
-    name: 'Keep writing summary',
+    name: "Keep writing summary",
     duration: 240
   },
   {
-    name: 'Some more web development',
+    name: "Some more web development",
     duration: 180
   },
   {
-    name: 'Staring out the window',
+    name: "Staring out the window",
     duration: 10
   },
   {
-    name: 'Talk to a lot of people',
+    name: "Talk to a lot of people",
     duration: 200
   },
   {
-    name: 'Look at application assignments new students',
+    name: "Look at application assignments new students",
     duration: 40
   }
 ];
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+let durationOfTasks = tasks.map(Element => Element.duration);
+
+ console.log(durationOfTasks);
+
+// Map the tasks to durations in hours.
+let durationInHours = durationOfTasks
+  .map(element => element / 60)
+
+ console.log(durationInHours);
+
+// Filter out everything that took less than two hours
+  
+let lessThanTwoHours = durationInHours
+    
+.filter(element => element >= 2)
+console.log(lessThanTwoHours);
+
+// Multiply the each duration by a per-hour rate for billing and sum it all up.
+
+let paymentPerHour = durationOfTasks
+  .map(element => (element / 60) * 100)
```

Comment:
The assignment states: _Each step should build on the result of the previous step._ This implies that you should use the **filtered** durations for the payment calculation.

I assume the number 100 is the hourly rate for Maartje (if that is the case, I want her job... oh wait, it is already taken).  Developers call such numbers that appear out of the blue without an obvious explanation 'magic' numbers. You should not use magic numbers because they make it hard to understand what is happening. Instead, define a named constant and use that throughout your code:

```js
const hourlyRate = 100;
```